Subpackages introspect and prop are broken with the go major module versioning

```go
conn, err := dbus.SystemBus()
if err != nil {
	return err
}
_, err = introspect.Call(conn.Object("org.freedesktop.DBus", "/"))
return err
```

Produces:

```
# command-line-arguments
./main.go:23:38: cannot use conn.Object("org.freedesktop.DBus", "/") (type "github.com/godbus/dbus/v5".BusObject) as type "github.com/godbus/dbus".BusObject in argument to introspect.Call:
        "github.com/godbus/dbus/v5".BusObject does not implement "github.com/godbus/dbus".BusObject (wrong type for Call method)
                have Call(string, "github.com/godbus/dbus/v5".Flags, ...interface {}) *"github.com/godbus/dbus/v5".Call
                want Call(string, "github.com/godbus/dbus".Flags, ...interface {}) *"github.com/godbus/dbus".Call
```

--- a/.travis.yml
+++ b/.travis.yml
@@ -3,9 +3,9 @@ language: go
 go_import_path: github.com/godbus/dbus
 
 go:
-  - 1.10.x
   - 1.11.x
   - 1.12.x
+  - 1.13.x
   - tip
 
 env:
@@ -38,6 +38,7 @@ addons:
     - dbus-x11
 
 before_install:
+  - export GO111MODULE=on
 
 script:
   - go test -v -race -mod=readonly ./...     # Run all the tests with the race detector enabled

--- a/.travis.yml
+++ b/.travis.yml
@@ -40,5 +40,5 @@ addons:
 before_install:
 
 script:
-  - go test -v -race ./...                   # Run all the tests with the race detector enabled
+  - go test -v -race -mod=readonly ./...     # Run all the tests with the race detector enabled
   - go vet ./...                             # go vet is the official Go static analyzer

--- a/_examples/bluetooth_introspect.go
+++ b/_examples/bluetooth_introspect.go
@@ -4,7 +4,7 @@ import (
 	"fmt"
 	"os"
 
-	"github.com/godbus/dbus"
+	"github.com/godbus/dbus/v5"
 )
 
 func main() {

--- a/_examples/eavesdrop.go
+++ b/_examples/eavesdrop.go
@@ -2,8 +2,9 @@ package main
 
 import (
 	"fmt"
-	"github.com/godbus/dbus"
 	"os"
+
+	"github.com/godbus/dbus/v5"
 )
 
 func main() {

--- a/_examples/introspect.go
+++ b/_examples/introspect.go
@@ -2,9 +2,10 @@ package main
 
 import (
 	"encoding/json"
-	"github.com/godbus/dbus"
-	"github.com/godbus/dbus/introspect"
 	"os"
+
+	"github.com/godbus/dbus/v5"
+	"github.com/godbus/dbus/v5/introspect"
 )
 
 func main() {

--- a/_examples/list-names.go
+++ b/_examples/list-names.go
@@ -2,8 +2,9 @@ package main
 
 import (
 	"fmt"
-	"github.com/godbus/dbus"
 	"os"
+
+	"github.com/godbus/dbus/v5"
 )
 
 func main() {

--- a/_examples/notification.go
+++ b/_examples/notification.go
@@ -1,6 +1,6 @@
 package main
 
-import "github.com/godbus/dbus"
+import "github.com/godbus/dbus/v5"
 
 func main() {
 	conn, err := dbus.SessionBus()

--- a/_examples/prop.go
+++ b/_examples/prop.go
@@ -2,10 +2,11 @@ package main
 
 import (
 	"fmt"
-	"github.com/godbus/dbus"
-	"github.com/godbus/dbus/introspect"
-	"github.com/godbus/dbus/prop"
 	"os"
+
+	"github.com/godbus/dbus/v5"
+	"github.com/godbus/dbus/v5/introspect"
+	"github.com/godbus/dbus/v5/prop"
 )
 
 type foo string

--- a/_examples/server.go
+++ b/_examples/server.go
@@ -2,9 +2,10 @@ package main
 
 import (
 	"fmt"
-	"github.com/godbus/dbus"
-	"github.com/godbus/dbus/introspect"
 	"os"
+
+	"github.com/godbus/dbus/v5"
+	"github.com/godbus/dbus/v5/introspect"
 )
 
 const intro = `

--- a/_examples/signal.go
+++ b/_examples/signal.go
@@ -2,8 +2,9 @@ package main
 
 import (
 	"fmt"
-	"github.com/godbus/dbus"
 	"os"
+
+	"github.com/godbus/dbus/v5"
 )
 
 func main() {

--- a/_examples/tcp_conn.go
+++ b/_examples/tcp_conn.go
@@ -5,7 +5,7 @@ import (
 	"net"
 	"os"
 
-	"github.com/godbus/dbus"
+	"github.com/godbus/dbus/v5"
 )
 
 // In order to enable TCP connections add the following configuration

--- a/introspect/call.go
+++ b/introspect/call.go
@@ -4,7 +4,7 @@ import (
 	"encoding/xml"
 	"strings"
 
-	"github.com/godbus/dbus"
+	"github.com/godbus/dbus/v5"
 )
 
 // Call calls org.freedesktop.Introspectable.Introspect on a remote object

--- a/introspect/introspectable.go
+++ b/introspect/introspectable.go
@@ -5,7 +5,7 @@ import (
 	"reflect"
 	"strings"
 
-	"github.com/godbus/dbus"
+	"github.com/godbus/dbus/v5"
 )
 
 // Introspectable implements org.freedesktop.Introspectable.

--- a/prop/prop.go
+++ b/prop/prop.go
@@ -5,8 +5,8 @@ package prop
 import (
 	"sync"
 
-	"github.com/godbus/dbus"
-	"github.com/godbus/dbus/introspect"
+	"github.com/godbus/dbus/v5"
+	"github.com/godbus/dbus/v5/introspect"
 )
 
 // EmitType controls how org.freedesktop.DBus.Properties.PropertiesChanged is

